src/CommunityToolkit/Xamarin.CommunityToolkit/Views/ViewSwitcher/ImageSwitcher.shared.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.UI.Views
+{
+	// TODO: Make it public once Android Renderer is implemeted
```

Comment:
Will we implement Android renderer on this PR?

Reply:
I am not sure because it requires time